pages/implementations/overview.md

```diff
@@ -0,0 +1,122 @@
+- `rulePage`: is the page containing a detailed rule description.
+
+```json
+ "a11y-testcases": [
+    {
+      "url": "https://auto-wcag.github.io/auto-wcag/auto-wcag-testcases/assets/SC1-1-1-image-has-name_passed_example_1.html",
+      "expected": "passed",
+      "ruleId": "SC1-1-1-image-has-name",
+      "rulePage": "https://auto-wcag.github.io/auto-wcag/rules/SC1-1-1-image-has-name.html"
+    },
+    ...
+ ]
+```
+
+The above test cases can be consumed to run against a test tool. The generated results are welcome to be submitted as implementation to the auto-wcag community.
+
+
+# {{ site.title }} Implementation
+
+Submitting implementations from various test tools, helps not only showcase the coverage of test cases authored for the rule, but to build a robust understanding of the WCAG rules and the respective implementations.
+
+Any test tool authors are welcome to submit the implementation results for the above test cases to the {{site.title}} community. 
+The implementation manifest should be a valid [JSON-LD](https://json-ld.org/spec/latest/json-ld/) document, where the assertions are expressed using [EARL](https://www.w3.org/TR/EARL10-Schema/).
+The manifest contains a list of assertion results for each of the test cases. 
+
+An example assertion for a test case, is as below:
+
+```json
+[
+  {
+    "@context": {
+      "doap": "http://usefulinc.com/ns/doap#",
+      "foaf": "http://xmlns.com/foaf/spec/#",
+      "@vocab": "http://www.w3.org/ns/earl#",
+      "earl": "http://www.w3.org/ns/earl#",
+      "WCAG20": "http://www.w3.org/TR/WCAG20/#",
+      "WCAG21": "http://www.w3.org/TR/WCAG21/#",
+      "auto-wcag": "https://auto-wcag.github.io/auto-wcag/rules/",
+      "dct": "http://purl.org/dc/terms#",
+      "sch": "https://schema.org/",
+      "WebPage": "sch:WebPage",
+      "url": "dct:source",
+      "assertions": {
+        "@reverse": "subject"
+      },
+      "assertedBy": {
+        "@type": "@id"
+      },
+      "outcome": {
+        "@type": "@id"
+      },
+      "mode": {
+        "@type": "@id"
+      },
+      "pointer": {
+        "@type": "ptr:CSSSelectorPointer"
+      }
+    },
+    "@type": "WebPage",
+    "url": "https://auto-wcag.github.io/auto-wcag/auto-wcag-testcases/assets/SC1-1-1-image-has-name_passed_example_1.html",
+    "assertions": [
+      {
+        "@type": "Assertion",
+        "mode": "earl:automatic",
+        "assertedBy": {
+          "@id": "==== YOUR TOOL ID & VERSION ====",
+          "@type": [
+            "earl:Assertor",
+            "earl:Software",
+            "doap:Project"
+          ],
+          "doap:name": "Axe",
+          "doap:vendor": {
+            "@id": "==== YOUR ORGANIZATION URL ====",
+            "@type": "foaf:Organization",
+            "foaf:name": "==== YOUR ORGANIZATION NAME ===="
+          }
+        },
+        "test": {
+          "@type": "TestCase",
+          "@id": "==== LINK TO RULE DESCRIPTION ===="
+        },
```

Comment:
If you add `"test": { "@type": "@id" }` to the context, you can just put a URL in test, instead of having to put this `@type` / `@id` object in here.